docs/pages/lunadefend/legacy/legacy-code.md

```diff
@@ -0,0 +1,23 @@
+---
+id: "legacy"
+title: "Legacy Information"
+sidebar_label: "Legacy Info"
+sidebar_position: 5
+---
+<!--
+  ~ Copyright by LunaSec (owned by Refinery Labs, Inc)
+  ~
+  ~ Licensed under the Creative Commons Attribution-ShareAlike 4.0 International
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License. You may obtain a copy of the License at
+  ~
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+
+## Naming Convention Changes
+
+The original LunaDefend project started out under the project name LunaSec, as the company has grown and the product matured, we decided to formalize the product stack as LunaDefend. Because of this change you may find sections of the code and commands still referencing the previous lunasec. We wanted to give context to any users or developers curious as to our naming references and conventions.
```

Comment:
Ah here we go. As I mentioned above, this isn't enough content for its own page, and its out of the way and the people that need to read it, aren't going to.  Theyre just going to be confused and muddle through.  Let's try to pinpoint that place people will be confused while reading our docs and put this there.

Reply:
Excellently worded though

Reply:
Maybe we just have a link to it from the pages where we start luna-sec

Reply:
So I'm thinking of just putting a :::note bubble at the bottom of the page with the text. And then I guess I'll just go ahead and remove the previous page.